schema_utils.py

```diff
@@ -387,3 +411,87 @@ def is_valid_email(obj):
             bool. Whether the given object is a valid email.
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
+
+    @staticmethod
+    def is_valid_expression(obj, algebraic=True):
```

Comment:
Per your other comment, this should probably become "is valid asciimath expression" (or the subset of asciimath that we support). The docstring for this method should explain clearly and fully what the definition of "validity" is.

Reply:
I've renamed the method and updated the docstring.